developer-docs/README.md

```diff
@@ -236,7 +236,10 @@ changes.
   PR, merge it and pull from the updated `main` branch. **It is mandatory to include this
   updated version of `main` in the subsequent steps.**
 
-1. Publish the wheel. Check if the generated webpage on `pypi.org` is rendered correctly.
+1. Publish the wheel. Check if the generated webpage on `pypi.org` is rendered correctly. 
+  If you have 2FA enabled for your account, you need to use an API Key as password and 
+  `__token__` as username when prompted by the CLI for credential. To set up a `pypi.org` 
+  API Key, go to "Account Settings" -> "API tokens".
```

Comment:
You can set up a config file to not have to worry about this:
```
$ cat ~/.pypirc
[pypi]
  username = __token__
  password = <the token>
```

Reply:
Done. Thanks!